illwill.nim

```diff
@@ -646,7 +950,6 @@ proc fill*(tb: var TerminalBuffer, x1, y1, x2, y2: Natural, ch: string = " ") =
       for x in x1..xe:
         tb[x, y] = c
 
-
```

Comment:
Please don't do this, please put all these empty lines back. They're there for a reason: when I use empty lines in a proc to make it better readable, then I use two lines to separate it from other procs. Python folks tend to do the same and I just like this rule, makes it easier to parse things visually.